<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="Vishwajit" failures="2" tests="2" name="Default test(failed)" time="0.081" errors="0" timestamp="10 Sep 2024 02:00:54 GMT">
  <testcase classname="ReTryTestCases.ReTestCase" name="TestCase1" time="0.02">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at ReTryTestCases.ReTestCase.TestCase1(ReTestCase.java:11)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TestCase1 -->
  <testcase classname="ReTryTestCases.ReTestCase" name="TestCase3" time="0.003">
    <failure type="junit.framework.AssertionFailedError">
      <![CDATA[junit.framework.AssertionFailedError
at junit.framework.Assert.fail(Assert.java:47)
at junit.framework.Assert.assertTrue(Assert.java:20)
at junit.framework.Assert.assertTrue(Assert.java:27)
at ReTryTestCases.ReTestCase.TestCase3(ReTestCase.java:23)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TestCase3 -->
  <testcase classname="ReTryTestCases.ReTestCase" name="TestCase2" time="0.0">
    <ignored/>
  </testcase> <!-- TestCase2 -->
</testsuite> <!-- Default test(failed) -->
